File: TaskManager/MyTaskManager/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyTaskManager
{
    public class Comment : ICloneable
    {
        public int Id { get; set; }
        public string Content { get; set; }
        public string Author { get; set; }
        public DateTime CreatedAt { get; set; }

        public Comment(int id, string content, string author)
        {
            Id = id;
            Content = content;
            Author = author;
            CreatedAt = DateTime.Now;
        }

        public object Clone()
        {
            return new Comment(Id, Content, Author)
            {
                CreatedAt = this.CreatedAt
            };
        }
    }
}

File: IssueTracker-Final/src/Comment.java
import java.time.LocalDateTime;

public class Comment {

    public int Id;

    public String Content;

    public String Author;

    public LocalDateTime CreatedAt;


    public Comment (int ID, String Content, String Author, LocalDateTime CreatedAt){

        this.Id=ID;
        this.Content=Content;

        this.Author=Author;

        this.CreatedAt=CreatedAt;

    }










}
